data/mods/DinoMod/monsters/dinosaur.json

```diff
@@ -1563,7 +1563,7 @@
     "harvest": "dino_feather_leather",
     "anger_triggers": [ "HURT", "FRIEND_ATTACKED", "PLAYER_NEAR_BABY" ],
     "fear_triggers": [ "PLAYER_CLOSE", "FRIEND_DIED" ],
-    "categories": [ "DINOSAUR", "WILDLIFE" ]
+    "categories": [ "DINOSAUR", "WILDLIFE", "MAN_MADE" ]
```

Comment:
Are oryctodromeus meant to be MAN_MADE? As far as I can tell, they are natural, or am I missing something?

Reply:
No, looks like I tagged the wrong dino, thanks!